stable/docker-registry/values.yaml

```diff
@@ -54,6 +54,14 @@ secrets:
 #   s3:
 #     accessKey: ""
 #     secretKey: ""
+
+# Options for s3 storage type:
```

Comment:
Add these correctly indented under the above s3 options.

Reply:
Above options are secrets, while these are not. Do you want all S3 options be under the secrets key?

Reply:
Sorry, my bad.